ocaml/ocaml-ppx_import/Portfile

```
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           ocaml 1.1
+PortGroup           github 1.0
+
+name                ocaml-ppx_import
+version             1.10.0
+revision            0
+categories          ocaml devel
+maintainers         {pguyot @pguyot} openmaintainer
+license             MIT
+
+github.setup        ocaml-ppx ppx_import ${version}
```

The `version` line above should be removed and the version should be specified here.
```suggestion
github.setup        ocaml-ppx ppx_import 1.10.0
```
Typically the `github.setup` line appears in the Portfile at the place where the `name` and `version` would otherwise appear. If the `name` needs to be changed, it is typically done on a subsequent line.

---

Thank you. I did update pending PRs accordingly.